packages/esm-service-queues-app/src/queue-entry-table-components/queue-priority.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { DefinitionTooltip, Tag } from '@carbon/react';
+import styles from './queue-priority.scss';
+import { type ConfigObject } from '../config-schema';
+import { type Concept } from '../types';
+import { useConfig } from '@openmrs/esm-framework';
+
+interface QueuePriorityProps {
+  priority: Concept;
+  priorityComment?: string;
+}
+
+const QueuePriority: React.FC<QueuePriorityProps> = ({ priority, priorityComment }) => {
+  const { priorityConfigs } = useConfig<ConfigObject>();
+  const priorityConfig = priorityConfigs?.find((c) => c.conceptUuid === priority.uuid);
+  return (
+    <>
+      {priorityComment ? (
+        <DefinitionTooltip className={styles.tooltip} align="bottom-left" definition={priorityComment}>
```

Comment:
Don't need to address it in this PR, but using tooltip means it won't work for mobile/tablet. We'll need better UX guidance on this.

Reply:
Yep.  We can ticket that separately as needed.  My goal here was to maintain existing behavior, just do so consistently